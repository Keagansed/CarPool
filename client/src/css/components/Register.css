#inputFirstName, 
#inputLastName,
#inputID,
#inputEmail
{
    background-color: rgba(0,0,0,0) !important;
    color: white;
    border: 0.05rem solid #6FEDB7;
}

/* Change Autocomplete styles in Chrome*/
input:-webkit-autofill,
input:-webkit-autofill:hover,
input:-webkit-autofill:focus {
    -webkit-text-fill-color: white;
    transition: background-color 5000s ease-in-out 0s;
} 