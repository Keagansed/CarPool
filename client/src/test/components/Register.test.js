import React from 'react';
import Register from '../../js/components/register/RegisterPage';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer'
import configureStore from 'redux-mock-store'

describe('Register Component', () => {
    const mockStore = configureStore();
    let store, container;
    
    beforeEach(() => {
        store = mockStore();
        container = shallow(<Register store={store} />);
    });
    
    it('captures snapshot', () => {
        const renderedValue = renderer.create(<Register store={store} />).toJSON();
        expect(renderedValue).toMatchSnapshot();
    });
  
    it('renders correctly', () => {
        expect(container.length).toEqual(1);
    });
});