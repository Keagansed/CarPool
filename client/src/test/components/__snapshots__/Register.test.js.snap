// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Register Component captures snapshot 1`] = `
<div>
  <div
    className="text-center"
  >
    <div
      className="container"
    >
      <div
        className="row bigTopMargin"
      >
        <div
          className="col-md-12"
        >
          <form
            className=""
          >
            <div
              className="form-group my-2"
            >
              <input
                className="form-control roundCorners bg-info border-1_5 border-primary w-75 mx-auto text-white"
                id="txtFirstName"
                name="txtFirstName"
                onChange={[Function]}
                placeholder="First Name"
                required="required"
                type="text"
              />
            </div>
            <div
              className="form-group"
            >
              <input
                className="form-control roundCorners bg-info border-1_5 border-primary w-75 mx-auto text-white"
                id="txtLastName"
                name="txtLastName"
                onChange={[Function]}
                placeholder="Last Name"
                required="required"
                type="text"
              />
            </div>
            <div
              className="form-group"
            >
              <input
                className="form-control roundCorners bg-info border-1_5 border-primary w-75 mx-auto text-white"
                id="txtID"
                name="txtID"
                onChange={[Function]}
                placeholder="ID Number"
                required="required"
                type="text"
              />
            </div>
            <div
              className="form-group"
            >
              <input
                className="form-control roundCorners bg-info border-1_5 border-primary w-75 mx-auto text-white"
                id="txtEmail"
                name="txtEmail"
                onChange={[Function]}
                placeholder="Email"
                required="required"
                type="email"
              />
            </div>
            <div
              className="form-group"
            >
              <input
                className="form-control roundCorners bg-info border-1_5 border-primary w-75 mx-auto text-white"
                id="txtPassword"
                name="txtPassword"
                onChange={[Function]}
                placeholder="Password"
                required="required"
                type="password"
              />
            </div>
            <div
              className="form-group"
            >
              <input
                className="form-control roundCorners bg-info border-1_5 border-primary w-75 mx-auto text-white"
                id="txtConfirmPassword"
                name="txtConfirmPassword"
                onChange={[Function]}
                placeholder="Confirm Password"
                required="required"
                type="password"
              />
            </div>
            <button
              className="mt-3 mb-2 btn btn-primary roundCorners w-75 text-secondary font-weight-medium"
              onClick={[Function]}
              type="submit"
            >
              <b>
                Register
              </b>
            </button>
          </form>
        </div>
      </div>
    </div>
  </div>
  <div
    className="py-2 text-center"
  >
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="col-md-12"
        >
          <strike>
                                
          </strike>
              
          <b
            className="small font-weight-medium"
          >
            OR
          </b>
              
          <strike>
                                
          </strike>
        </div>
      </div>
    </div>
  </div>
  <div
    className="my-2"
  >
    <div
      className="text-center container"
    >
      <div
        className="row text-center"
      >
        <div
          className="col-2-pr-5 col-sm-2 col-md-6 col-lg-2 col-xl-2 ml-auto"
        >
          <img
            alt="fbLogin"
            className="img-fluid d-block mx-auto"
            id="btnFacebookLogin"
            src="fb_icon.png"
          />
        </div>
        <div
          className="col-2-pl-5 col-sm-2 col-md-6 col-lg-2 col-xl-2 mr-auto"
        >
          <img
            alt="googleLogin"
            className="img-fluid d-block mx-auto"
            id="btnGoogleLogin"
            src="google_icon.png"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
